src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -47,6 +48,27 @@ def _block_version(self, block, version, block_type=BlockType.BLOCKED):
         )
 
 
+class TestOrderedDiffLists(_MLBFBase):
```

Comment:
Don't need to depend on `_MLBFBase`, it could just be `TestCase` (or `SimpleTestCase` since you don't do database stuff, though we tend not to use that one)